packages/components-js/src/examples/pin-code-example.html

```diff
@@ -0,0 +1,16 @@
+<form>
+  <p-pin-code label="Some Label" name="pin-code"></p-pin-code>
+  <p-button type="submit">Submit</p-button>
+</form>
+
+<p-text>Last submitted data: none</p-text>
+
+<script>
+  const debugElement = document.querySelector('p-text');
+  const form = document.querySelector('form');
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const formData = new FormData(e.target);
+    debugElement.innerText = `Last submitted data: ${Array.from(formData.values()).join(', ') || 'none'}`;
```

Comment:
```suggestion
    debugElement.innerText = `Last submitted data: ${formData.values() || 'none'}`;
```
I think you don't need the join since its not an array?

Reply:
value returns an iterator, that's why it is needed here